### PR TITLE
DEV-15556 added ScheduleWorkTimeResolver

### PR DESCRIPTION
**Pull request by olexandrpolosmak**

### CODE REVIEW

File: src/Dots/WorkTimeSchedule/WorkTimeSchedule.php [hunk 1]
```diff
@@ -0,0 +1,277 @@
+<?php
+
```
Comment (olexandrpolosmak) [inline src/Dots/WorkTimeSchedule/WorkTimeSchedule.php:2]: lets:
 - remove bool, I suggest to use null
 - what about isAlwaysOn() ? Just I need to use him in many places
 - there is a bug, current logic expects that the schedule has whole seven days

File: src/Dots/WorkTimeSchedule/WorkTimeSchedule.php [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+namespace Dots\WorkTimeSchedule;
+
+use Dots\Data\Entity;
+use Exception;
+use Illuminate\Support\Carbon;
+use RuntimeException;
+
+class WorkTimeSchedule extends Entity
+{
+    protected bool $alwaysOn = false;
```
Comment (yehorherasymchuk) [inline src/Dots/WorkTimeSchedule/WorkTimeSchedule.php]: alwaysOn  => anytime = true/false

Comment (yehorherasymchuk) [reply src/Dots/WorkTimeSchedule/WorkTimeSchedule.php]: timezone not nullable

File: src/Dots/WorkTimeSchedule/WorkTimeSchedule.php [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    }
+
+    public function getWorkTimeToday(): ?array
+    {
+        $workTime = $this->getSchedule();
+
+        return $workTime[$this->getCurrentWeekDay()] ?? null;
+    }
+
+    public function getStartTimeToday(): int|bool
```
Comment (yehorherasymchuk) [inline src/Dots/WorkTimeSchedule/WorkTimeSchedule.php]: use nullable

**Review by yehorherasymchuk — CHANGES_REQUESTED** [review]

**Review by yehorherasymchuk — APPROVED** [review]

**Pull request merged**